internal/icinga2/client_api.go

```diff
@@ -212,17 +213,28 @@ func (client *Client) fetchCheckable(ctx context.Context, host, service string)
 	return &objQueriesResults[0], nil
 }
 
+// fetchAcknowledgementCommentNoComment is an error indicating that no Comment for an Acknowledgement exists.
+//
+// This error should only be wrapped and returned from the fetchAcknowledgementComment method and only if no Comment was
+// found. For other errors, like network errors, this error must not be used.
+var fetchAcknowledgementCommentNoComment = errors.New("found no Acknowledgement Comment")
```

Comment:
Something less convoluted should do as variable name as well, something like `missingAcknowledgeComment`.

Reply:
Please also prefix it with an `err`.